resources/authgear/templates/en/translation.json

```diff
@@ -942,6 +942,7 @@
   "v2.component.verify-bot-protection.default.title": "Checking your system...",
   "v2.error.account-conflict": "{ IdentityTypeIncoming, select, login_id {{FlowType, select, login {Looks like you used {OAuthProviderNameExisting} for login before.} other {This {LoginIDNameIncoming} is already registered.}}} oauth {Looks like you haven''t linked up with that {OAuthProviderNameIncoming} account yet.} other {This account is already registered} }",
   "v2.error.account-not-found": "{ IdentityTypeIncoming, select, login_id {{ LoginIDTypeIncoming, select, email {This email} phone {This phone number} username {This username} other {This identity} }} oauth {This email} other {This identity} } cannot be found. Do you want to create an account?",
+  "v2.error.secondary-authenticator-required": "At least one 2-factor authentication method is required, please add another method before removing this one.",
```

Comment:
We have v2.error.remove-last-secondary-authenticator already. Let's reuse that?

Reply:
Oh doesn't know that. Maybe I should update the reason to `RemoveLastSecondaryAuthenticator` too.

Reply:
No. Use the new error please.